deploy/olm-catalog/managed-api-service/managed-api.package-service.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
+- currentCSV: managed-api-service.v0.0.0
```

Comment:
Interesting that v0.0.0 versioning works. Will we merge it with this?

Reply:
The plan was to have v0.0.0 there as a place holder till we were ready to do a release. Sense you are the second person to ask this maybe we should readdress this plan.